### DISCUSSION TITLE
感谢程序员们，技术引导革命

### DISCUSSION THREAD

**Discussion by freedom1933** [WeNeedHome/SummaryOfLoanSuspension#918] [Ideas]
历史上，革命或改革，都是由政治家主导的，但政治家基本都是蛊王。
激烈的权力斗争，让政治异化了或妖魔化了。
政治本质是什么？其实就是人与人的博弈。
博弈，本身是中性的，可以光明正大，可以在普世价值下权衡，甚至，长期博弈的最优策略，就是以血还血，以牙还牙，非常符合最朴素的道德观。
但从古至今，博弈过程的信息丢失，让阴暗和邪恶大行其道，让善恶无报，让劣币驱逐良币......
现代信息技术，让阴暗在逐渐消失，让民众记忆变得更加长久和全面。
因此，非常感谢信息技术工作者，联合起来吧，你们将引导人类文明的进步，替代蛊王们！！！

**Comment by daxiazilong** [comment]
1001011101101001x100111101100000x1001111010111011x111010111111001

**Comment by nijisakai** [comment]
我们在这里应该感谢美国

**Reply by freedom1933** [reply]
现代信息技术的创建和发展，基本上都是欧美的贡献，因此，确实应该感谢欧美~~
